src/app/core/pricing/pricing-tier.service.ts

```diff
@@ -5,6 +5,8 @@ import { BehaviorSubject, Observable } from "rxjs";
     providedIn: "root"
 })
 export class PricingTierService {
+    private static ITEM_COUNT_THRESHOLD: number = 4;
```

Comment:
instead of use a static property, just use a const variable defined outside the class